README.md

```diff
@@ -1,2 +1,105 @@
 # terraform-provider
 Ably Terraform Provider
+
+# [Ably](https://www.ably.com)
+
+_[Ably](https://ably.com) is the platform that powers synchronized digital experiences in realtime. Whether attending an event in a virtual venue, receiving realtime financial information, or monitoring live car performance data – consumers simply expect realtime digital experiences as standard. Ably provides a suite of APIs to build, extend, and deliver powerful digital experiences in realtime for more than 250 million devices across 80 countries each month. Organizations like Bloomberg, HubSpot, Verizon, and Hopin depend on Ably’s platform to offload the growing complexity of business-critical realtime data synchronization at global scale. For more information, see the [Ably documentation](https://ably.com/docs)._
+
+[![npm version](https://img.shields.io/npm/v/ably.svg?style=flat)](https://img.shields.io/npm/v/ably.svg?style=flat)
+
+This is a Terraform provider for Ably.
```

Comment:
I would expand this paragraph to include some detail around what this means for a user... including key terms/phrases and a bit of fizz - e.g. "first class support", "infrastructure as code", "conformed deployment stack", etc..

Reply:
Updated in b233642b38be398f42597fe5881fd1ead2e3e8ef - wdyt?